docs/documentation.md

```diff
@@ -0,0 +1,56 @@
+# Documentation in MDS
+
+Document, document, document!
+
+1. [Purpose](#purpose)
+1. [Accessibility](#accessibility)
+1. [Examples](#examples)
+1. [Template](#template)
+
+## Purpose
+
+A short blurb on why the component exists.
+This helps define the desired responsibilities of the component.
+
+## Accessibility
+
+In order to maintain a11y as a first-class design principle,
+each component requires an a11y section.
+The section describes the keyboard support, mouse support, and label support.
+
+## Examples
+
+The code examples should be shareable.
+
+1. Always import the components used in the example
```

Comment:
Reasoning:

- Also, always add `@mavenlink/design-system` to import paths. e.g. `import Buttom from '@mavenlink/design-system/src/components/button/button.jsx'`
- Copy-pastable
- Bigmaven auto-documentation can actually render it

Reply:
We might want a linter for this